### ISSUE TITLE
[ENHANCEMENT] Compare text and sync

### ISSUE THREAD

**Issue by gabriellluz** [SubtitleEdit/subtitleedit#2161]
Here's the scenario:

I got a Japanese transcript for a japanese drama tv show and real English subtitle for the same tv show.
The japanese transcript is NOT timed and the English subtitle is actually timed.

So:
Japanese transcript:
NO TIMING
ちがうよ。藤岡のクラスに留学生、来たんだって？

English subtitles for the same sentence above (human translation):
1
00:02:17,440 --> 00:02:20,375
That's not it. I heard an exchange student came to your class.

Translated Japanese transcript with Google Translate:
NO TIMING
It is different. Did you come to Fujioka class as an exchange student?

So I have the japanese transcript and I don't want to stay hours and hours copying and pasting Japanese subs on Subtitle Edit. What I want to do is:
- use the English subtitle and compare with the translated Japanese transcript sentence;
- if it has at least 2 or more exact words, then it gets the timing for the English subtitles.
At the end we'll have all or most of the Japanese subtitles done using the same timing for the English subtitles. The reviewing process would be much shorter. I usually sync Japanese subtitles so I can watch Japanese movies with Japanese subtitles.

We'd have to use something like:
1
00:02:17,440 --> 00:02:20,375
ちがうよ。藤岡のクラスに留学生、来たんだって？
It is different. Did you come to Fujioka class as an exchange student?

Since SubtitleEdit would actually sync the Japanese transcript with the English subtitles, I'd just have to remove the second line. It would be like this:

1
00:02:17,440 --> 00:02:20,375
ちがうよ。藤岡のクラスに留学生、来たんだって？

Basically, the tool would just compare the subtitles and sync them purely based on the text itself.

**Comment by gabriellluz** [comment]
I'll leave it closed for now.
